zerver/lib/export.py

```diff
@@ -1458,6 +1461,12 @@ def do_import_realm(import_dir: Path) -> Realm:
     convert_to_id_fields(data, 'zerver_realm', 'notifications_stream')
     fix_datetime_fields(data, 'zerver_realm')
     fix_realm_authentication_bitfield(data, 'zerver_realm', 'authentication_methods')
+    realm_id_list = current_table_ids(data, 'zerver_realm')
+    allocated_realm_id_list = allocate_ids(Realm, len(data['zerver_realm']))
+    for item in range(len(data['zerver_realm'])):
+        update_id_map('realm', realm_id_list[item], allocated_realm_id_list[item])
```

Comment:
`update_id_map` Maps `ID` to `ID` relation, hence we pass it IDs from both  `realm_id_list` and `allocated_realm_id_list`